src/tests/EpicsClientMonitor_tests.cpp

```diff
@@ -0,0 +1,106 @@
+#include "../EpicsClient/EpicsClientMonitor.h"
+#include <gtest/gtest.h>
+#include <helper.h>
+
+using namespace Forwarder;
+
+TEST(EpicsClientMonitorTest, TestValueIsCachedWhenEmitIsCalledFirstTime) {
+  ChannelInfo ChannelInfo;
```

Comment:
The convention for test naming seems to be snake_case if only because most of the other tests are that way already ;)